### PR TITLE
feat: zwave-js v10 support

### PR DESCRIPTION
**Pull request by robertsLando**

### CODE REVIEW

File: lib/ZwaveClient.ts [hunk 1]
```diff
@@ -65,6 +65,9 @@ import {
 	SerialAPISetupCommand,
 	FirmwareUpdateFileInfo,
 	FirmwareUpdateInfo,
```
Comment (AlCalzone) [inline lib/ZwaveClient.ts:67]: You got a linter warning here

Comment (AlCalzone) [reply lib/ZwaveClient.ts:67]: 'FirmwareUpdateInfo' is defined but never used

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: lib/ZwaveClient.ts (hunk 1 above)
Comment (robertsLando) [reply lib/ZwaveClient.ts]: fixed

**Comment by robertsLando** [comment]
@AlCalzone I have also introduced `node found` event, when you add a node it should be immediatly shown in control panel as soon as it is found

**Comment by AlCalzone** [comment]
That seems to work, but is hard to test

**Review by AlCalzone — DISMISSED** [review]

**Pull request merged**